spec/support/capybara.rb

```diff
@@ -0,0 +1,37 @@
+def take_screenshot
+  unless ENV.key?('CIRCLECI')
+    page.save_screenshot Rails.root.join('tmp', 'capybara', "screenshot-#{DateTime.now.strftime('%Y%m%d%H%M%S')}.png")
```

Comment:
__[RuboCop]__ Rails/TimeZone: Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.xmlschema`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.httpdate`, `DateTime.now.to_i`, `DateTime.now.to_f` instead. ([link](https://sider.review/gh/repos/47168038/pulls/537#issue-3998734))

<sub>posted by Sider</sub>